Form-Controls/index.html

```diff
@@ -1,27 +1,69 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-  <meta charset="utf-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <title>My form exercise</title>
-  <meta name="description" content="">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="stylesheet" href="styles.css">
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <title>Form exercise</title>
+  <meta name="description"
+    content="The form collects client details about their purchase, item characteristics, and delivery information, including the address for shipping." />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <link rel="stylesheet" href="styles.css" />
 </head>
+
 <body>
   <header>
-    <h1>Product Pick</h1>
+    <h1>ZARA STORES</h1>
   </header>
   <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
+    <div class="container">
+      <form id="form" class="form">
+        <h2>Please choose the size and color of your  T-shirt</h2>
+        <div class="form-control">
+          <label for="username">User</label>
+          <input type="text" id="username" name="username" placeholder="Enter Username" minlength="2" required />
+        </div>
+        <div class="form-control">
+          <label for="email" class="email">Email</label>
+          <input type="email" id="email" placeholder="Enter your email" pattern="[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*" required />
+        </div>
+        <div class="form-control">
+          <label for="password">Password</label>
+          <input type="password" id="password" placeholder="Enter Password" required />
+        </div>
+        <div class="form-size">
+          <h3>Please pick your color</h3>
+          <input id="red" type="radio" name="color" value="red" required />
+          <label for="red">Red</label>
+
+          <input id="yellow" type="radio" name="color" value="yellow" />
+          <label for="yellow">Yellow</label>
 
+          <input id="blue" type="radio" name="color" value="blue" />
+          <label for="blue">Blue</label>
+        </div>
+        <div class="dropdown">
+          <label for="size"><b>Please select your size:</b></label>
+          <select id="size" name="size" required>
+            <option value="" disabled selected>Here</option>
+            <option value="XS">XS</option>
+            <option value="S">S</option>
+            <option value="M">M</option>
+            <option value="L">L</option>
+            <option value="XL">XL</option>
+            <option value="XXL">XXL</option>
+          </select>    
+        </div>
+        <div class="form-delivery">
+          <label for="delivery-date"><b>Please select your Delivery Date between the following 4 weeks.</b></label>
+          <input type="date" name="delivery-date" min="2024-11-18" max="2024-12-14">
```

Comment:
Hi @Grajales-K ,
Is there a reason for not adding an id attribute to the input element for the delivery-date?

Reply:
I overlooked it; my mistake. I’ve made the update now. Thank you!